programs/utilities/block_counter.sc

```diff
@@ -1,28 +1,147 @@
+//Counts blocks in a given region, optionally filtering by block type or tag. 
+//Prints the output into a chat or a file.
+//If vertical scan is set to true, it will return a separate report for each y value.
+//By gnembom and Firigion
```

Comment:
Misspelled gnomgnom

Reply:
my bad

Reply:
```suggestion
//By gnembon and Firigion
```

Reply:
Oh, I thought it was a joke. sorry, i really don't see that sort of typos.